### ISSUE TITLE
Per-user exceptions to MOVE_SPAM_TO_JUNK=1

### ISSUE THREAD

**Issue by frugan-it** [docker-mailserver/docker-mailserver#2319]
Is it possible to set exceptions to `MOVE_SPAM_TO_JUNK=1`?

I use these 2 variables:
```
SPAMASSASSIN_SPAM_TO_INBOX=1
MOVE_SPAM_TO_JUNK=1
```
and everything works fine, the spam is delivered to the Junk folder.

However, I would like to set exceptions for some clients who are used to connecting only in POP3, and therefore do not download the Junk folder.

I tried to create this file `mailserver/data/example.org/someone/.dovecot.sieve`:

```
require "fileinto";

if header :contains "X-Spam-Flag" "YES" {
  fileinto "INBOX";
}
```

but the rules are completely ignored, even though they have been compiled (the `.dovecot.svbin` file is present).

I assume at this point I should try to move the rules to the file `mailserver/config/before.dovecot.sieve`, right?
Eg. something like that:

```
require ["fileinto", "mailbox"];

if address "To" "someone@example.org" {
  if header :contains "X-Spam-Flag" "YES" {
    fileinto "INBOX";
  }
}
```
But I wanted to know if my approach is correct, or if there is a workaround.

Thanks!

**Comment by georglauterbach** [comment]
> Is it possible to set exceptions to `MOVE_SPAM_TO_JUNK=1`?

No, at least none that I know of. As for the other Dovecot related questions, I cannot really help you as I'm not into POP3 and how to filter on that with Dovecot.
